components/common/TextConstants.qml

```diff
@@ -31,16 +31,23 @@ QtObject {
     readonly property string loginFailed:       qsTr("Login failed")
     readonly property string loginSucceeded:    qsTr("Login succeeded")
     readonly property string password:          qsTr("Password")
+    readonly property string emptyPassword:     qsTr("Input empty. Please enter password!")
+    readonly property string passwordChange:    qsTr("Change password")
     readonly property string prompt:            qsTr("Enter your username and password")
     readonly property string promptSelectUser:  qsTr("Select your user and enter password")
     readonly property string promptUser:        qsTr("Enter your username")
     readonly property string promptPassword:    qsTr("Enter your password")
+    readonly property string emptyPrompt:       qsTr("Empty prompt from pam")
```

Comment:
Not sure we want to mention pam in a user message?

Reply:
Yepp, you are right, replace it with ("Please enter a password")?

Reply:
When is this message seen?
If it somehow related to changing the password I would say the latter, otherwise the former.

Reply:
I think i will go with
```qml
readonly property string emptyPrompt:       qsTr("unregistered user")
```
, if I understand the code right.
I asked the author if that is okay
See comment https://github.com/sddm/sddm/pull/776#issuecomment-538627725

Reply:
done,
https://github.com/sddm/sddm/pull/1121/commits/dadb3df75067155a355679060c4e64f25c21ce4f
https://github.com/sddm/sddm/pull/1121/commits/5642c871304808c694a91dec4f0a9a69d2474593
https://github.com/sddm/sddm/pull/1121/commits/79cf8f12c61f4def13b611db752660df566a9085